tools/lib/git.sh

```diff
@@ -90,3 +90,13 @@ git_base_commit() {
 git_changed_files() {
     git diff --name-only --diff-filter=d "$@"
 }
+
+# Note that this assumes ONE submodule. If we ever have more than one,
+# we'll need to generalize this.
```

Comment:
This comment no longer applies, right? (After the revision corresponding to https://github.com/zulip/zulip-flutter/pull/579#discussion_r1540345664 .)